truss-chains/truss_chains/model_skeleton.py

```diff
@@ -16,11 +15,7 @@ class TrussChainletModel:
     _chainlet: definitions.ABCChainlet
 
     def __init__(
-        self,
-        config: dict,
-        data_dir: pathlib.Path,
-        secrets: secrets_resolver.Secrets,
-        environment: Optional[dict],
+        self, config: dict, data_dir: pathlib.Path, secrets: secrets_resolver.Secrets
```

Comment:
```suggestion
                environment: Optional[dict] = None,  # TODO: remove after version sync.
```

Reply:
ok good catch - will put up a pr again with this fix